zio-http/jvm/src/test/scala/zio/http/RoutePatternSpec.scala

```diff
@@ -399,12 +399,32 @@ object RoutePatternSpec extends ZIOHttpSpec {
         assertTrue(routePattern.format((1, "abc")) == Right(Path("/users/1/posts/abc")))
       },
     )
+  def anyOfTests =
+    suite("anyOf")(
+      test("anyOf with multiple patterns") {
+        val routePattern = Method.GET / anyOf("", "index.html", "index.htm")
+
+        println("Testing routePattern with empty path:")
+        println(routePattern.matches(Method.GET, Path("")))
+      
+        println("Testing routePattern with index.html path:")
+        println(routePattern.matches(Method.GET, Path("index.html")))
+      
+        println("Testing routePattern with index.htm path:")
+        println(routePattern.matches(Method.GET, Path("index.htm")))
+      
+        assertTrue(routePattern.matches(Method.GET, Path("")))
+        assertTrue(routePattern.matches(Method.GET, Path("index.html")))
+        assertTrue(routePattern.matches(Method.GET, Path("index.htm")))
```

Comment:
The first 2 are not being evaluated (assertions in `zio-test` need to be composed). Use this instead:

```suggestion
        assertTrue(
          routePattern.matches(Method.GET, Path("")),
          routePattern.matches(Method.GET, Path("index.html")),
          routePattern.matches(Method.GET, Path("index.htm"))
        )
```

Also probably a good idea to add a few assertions on a path that shouldn't be matching, e.g.,

```scala
!routePattern.matches(Method.GET, Path("foo"))
!routePattern.matches(Method.POST, Path("index.html"))
```